db/sql/table_partial.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS partial;
+CREATE TABLE partial
```

Comment:
Update the seeder to create a partial.

Reply:
I hadn't even considered the seeder, good catch.